tools/topology/topology1/development/sof-imx8-xtensa-src-wm8960.m4

```diff
@@ -0,0 +1,128 @@
+#
+# Topology with codec_adapter processing component for i.MX8QM/i.MX8QXP
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`sai.m4')
+include(`pcm.m4')
+include(`buffer.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include DSP configuration
+include(`platform/imx/imx8.m4')
+
+
+# Post process setup config
+
+# codec Post Process setup config
+#
+# Define the pipelines
+#
+# PCM0_P --> B0 --> CODEC_ADAPTER --> B1 --> SAI1 (wm8960)
+#
+
+
+DECLARE_SOF_RT_UUID("Cadence Codec", cadence_codec_uuid, 0xd8218443, 0x5ff3,
```

Comment:
Do we need to say Cadence SRC here or is there > 1 processing component in the Cadence archive ?